testsuite/fuzzer/fuzz/fuzz_targets/move/aptosvm_publish_and_run.rs

```diff
@@ -184,6 +184,26 @@ fn run_case(mut input: RunnableState) -> Result<(), Corpus> {
     AptosVM::set_concurrency_level_once(2);
     let mut vm = FakeExecutor::from_genesis(&VM, ChainId::mainnet()).set_not_parallel();
 
+    // reject any TypeParameter exceeds the maximum allowed value (Avoid known Ivariant Violation)
```

Comment:
it would be cleaner/clearer to move this in an external function, maybe a generic "filter" function where we handle all false positive